mix.exs

```diff
@@ -20,19 +20,39 @@ defmodule Stripe.Mixfile do
   # Configuration for the OTP application
   def application do
     [
-      applications: [:httpoison]
+      env: env(),
+      applications: apps(Mix.env)
     ]
   end
 
+  defp env() do
+    [
+      stripity_stripe: [
+        api_version: "2016-07-06",
+        api_base_url: "https://api.stripe.com/v1/",
+        use_connection_pool: true,
+        pool_options: [
+          timeout: 5_000,
+          max_connections: 10
+        ]
+      ]
+    ]
+  end
+
+  defp apps(:test), do: [:bypass | apps()]
+  defp apps(_), do: apps()
+
+  defp apps(), do: [:hackney, :poison, :logger]
+
   defp deps do
     [
-      {:httpoison, ">= 0.0.0" },
-      {:poison, ">= 0.0.0", optional: true},
-      {:ex_doc, ">= 0.0.0", only: :dev},
-      {:earmark, ">= 0.0.0", only: :dev},
-      {:excoveralls, ">= 0.0.0", only: :test},
-      {:mock, ">= 0.0.0", only: :test},
-      {:inch_ex, ">= 0.0.0", only: [:dev, :test]}
+      {:hackney, "~> 1.6"},
+      {:poison, "~> 2.0"},
+      {:bypass, "~> 0.5", only: :test},
+      {:ex_doc, "~> 0.14", only: :dev},
+      {:earmark, "~> 1.0", only: :dev},
+      {:excoveralls, "~> 0.5", only: :test},
+      {:inch_ex, "~> 0.5", only: [:dev, :test]}
```

Comment:
Would just recommend alpha-reordering these, but that's personal preference for us.